modular_skyrat/modules/assault_operatives/code/turrets.dm

```diff
@@ -0,0 +1,32 @@
+//TURRETS>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
```

Comment:
God that's like the most annoying form of comment when I'm looking for merge conflict markers.
```suggestion
//TURRETS
```